MAINTAINERS

```diff
@@ -0,0 +1,14 @@
+# Org-Level Maintainers (in alphabetical order)
+# Pattern: [First Name] [Last Name] <[Email Address]> ([GitHub Handle])
+Justin Cormack <justin.cormack@docker.com> (@justincormack)
+Niaz Khan <niazfk@amazon.com> (@niazfk)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+
+# Repo-Level Maintainers (in alphabetical order)
```

Comment:
Should we mention repo names here?

Reply:
Good point, @priteshbandi! 

We could do that. The CODEOWNERS should automatically propagate through the repos. Though for the MAINTAINERS I don't think this will work.

Reply:
Re-reading the documentation it doesn't seem that we can leverage the CODEOWNERS file in the `.github` repo for the rest of the repositories. The "Repo-Level Maintainers" in this context means the `.github` repo only.

Reply:
sounds like a nice feature request, but bummer